pages/api/children/[childId]/accounts.js

```diff
@@ -0,0 +1,76 @@
+import connectDB from "../../_db/connect-db";
+import {Child} from "../../_db/models/Child";
+// import {Account} from "../../_db/models/Account";
+
+async function handler(req, res) {
+  switch (req.method) {
+    // case "GET":
+    //   try {
+    //     const filter = {};
+    //     if (req.query._id) {
+    //       filter._id = req.query._id;
+    //       // filter.lastName = req.query.lastName;
+    //       // filter.password = req.query.password;
+    //     }
+    //     const children = await Child.find(filter);
+    //     res.status(200).json(children[0]);
+    //   } catch (error) {
+    //     // You can inspect the error and return more meaningful error messages...
+    //     res.status(500).json({error: "something went wrong"});
+    //   }
+    //   break;
+    case "POST":
+      try {
+        const child = await Child.findById(req.query.childId);
+
+        if (child.accounts) {
+          child.accounts.push(req.body);
+        } else {
+          child.accounts = [req.body];
+        }
+        console.log(child.accounts);
+        console.log(req.body);
```

Comment:
Please delete this `console.log`s:

```suggestion
```

Reply:
As I am still working on this function, I still need these 2 `console.log()` but I will delete them when I am done.